kuma/test_urls.py

```diff
@@ -0,0 +1,82 @@
+import pytest
```

Comment:
I think these tests could live in ``kuma/core/tests/test_views.py``, where the ``events`` test ended up, rather than have a "naked" test file at this level.